agent/hibinit-agent

```diff
@@ -288,19 +300,19 @@ class BackgroundInitializerRunner(object):
     def do_async_init(self):
         try:
             self.swapper.init_swap()
-            update_kernel_swap_offset(self.swapper.swapon, self.swapper.swapoff, self.swapper.filename, self.update_grub)
+            update_kernel_swap_offset(self.swapper.swapon, self.swapper.swapoff, self.swapper.filename, self.update_grub, self.btrfs_enabled)
         except Exception as ex:
-            log("Failed to initialize swap, reason: %s" % str(ex))
+            log("Failed to initialize swap when using async, reason: %s" % str(ex))
```

Comment:
Please make sure to test it with older Amis in fedora

Reply:
Yes, will do, and will leave an update
p.s. not sure the comment is related to the code selected as this is just was just added to differentiate between two logs with the same output `Failed to initialize swap`. Taking it as a general comment

Reply:
Ran test on pre-btrfs amis with`btrfs-enabled=False` with 100% success

```
|  InstanceType  |     Platform     |RunsSucceeded|RunsFailed|RunsInputInvalid|RunsRetriable|RunsIced|Success Rate|
|      m7i.xlarge|            fedora|          100|         0|               0|            0|       0|     100.00%|
|     m7i.4xlarge|            fedora|          100|         0|               0|            0|       0|     100.00%|
|      m7i.medium|            fedora|          100|         0|               0|            0|       0|     100.00%|
|       m7i.large|            fedora|          100|         0|               0|            0|       0|     100.00%|
|     m7i.2xlarge|            fedora|          100|         0|               0|            0|       0|     100.00%|
```